packages/components/src/components/common/inputbadge/inputBadge.stories.tsx

```diff
@@ -44,4 +45,36 @@ export const BadgeArea: Story = {
   render: props => <InputBadgeArea {...props} />
 };
 
+export const InputBadgeDescription: Story = {
+  render: props => (
+    <BasicField message={{ message: 'this is a description', variant: 'description' }}>
+      <InputBadge {...props} />
+    </BasicField>
+  )
+};
+
+export const InputBadgeInfo: Story = {
+  render: props => (
+    <BasicField message={{ message: 'this is an info', variant: 'info' }}>
+      <InputBadge {...props} />
+    </BasicField>
+  )
+};
+
+export const InputBadgeWarning: Story = {
+  render: props => (
+    <BasicField message={{ message: 'this is a warning', variant: 'warning' }}>
+      <InputBadge {...props} />
+    </BasicField>
+  )
+};
+
+export const InputBadgeError: Story = {
+  render: props => (
+    <BasicField message={{ message: 'this is an error', variant: 'error' }}>
+      <InputBadge {...props} />
+    </BasicField>
+  )
+};
```

Comment:
IMO not needed to add a example for each variant.
One would be enough for me, maybe with the default setting warning or error and then define a storybook argument so you can switch it in the view.
E.g. like the Message/WithInput does it